src/Npgsql/NpgsqlConnector.cs

```diff
@@ -353,6 +359,9 @@ int InternalCommandTimeout
                 if (internalTimeout == -1)
                     return Math.Max(Settings.CommandTimeout, MinimumInternalCommandTimeout) * 1000;
 
+                // Todo: Decide what we really want here
```

Comment:
I'm getting pretty sure that we should kill the internal timeout feature altogether (#3045).

Reply:
I'd say that that's out of scope for this PR - no?